src/typeDefs/types.ts

```diff
@@ -29,7 +29,7 @@ export const types = gql`
   type Comment {
     _id: ID
     text: String!
-    createdAt: String
+    createdAt: Timestamp
```

Comment:
Don't use timestamp. Replace all instances with DateTime.

Reply:
Okay. Any specific reason for this though?

Reply:
@EshaanAgg They can't be used for localised date. You can't change them dynamically based on client's time zone and stuff.

Reply:
Ohh, wow. Wasn't aware about the same. Made the required changes in the latest commit.